packages/trpc/server/routers/viewer/availability/team/listTeamAvailability.schema.ts

```diff
@@ -6,6 +6,7 @@ export const ZListTeamAvailaiblityScheme = z.object({
   startDate: z.string(),
   endDate: z.string(),
   loggedInUsersTz: z.string(),
+  teamId: z.number().optional(),
```

Comment:
Will be used for filters in a follow up. Just wanted to keep this PR as minimal as possible